Cython/Utility/Exceptions.c

```diff
@@ -784,6 +784,12 @@ static void __Pyx_AddTraceback(const char *funcname, int c_line,
 
 #include "compile.h"
 #include "frameobject.h"
+#if PY_VERSION_HEX >= 0x030b00a6
+  #ifndef Py_BUILD_CORE
+    #define Py_BUILD_CORE 1
+  #endif
+  #include "internal/pycore_frame.h"
+#endif
 #include "traceback.h"
```

Comment:
Let's reduce the impact of `Py_BUILD_CORE` here.
```suggestion
#include "traceback.h"
#if PY_VERSION_HEX >= 0x030b00a6
  #ifndef Py_BUILD_CORE
    #define Py_BUILD_CORE 1
  #endif
  #include "internal/pycore_frame.h"
#endif
```